frontend/src/actions/Download.js

```diff
@@ -68,6 +82,11 @@ class Download extends Action {
                 .then(function (blob) {
                 saveAs(blob, directory[j].name+".zip");
                 });
+
+            // Progress bar for folders, check if files message is needed
+            msg = bothTypes ? msgStart+(j+1+"/")+folderCount+msgMiddle+0+"/"+filesCount : msgStart+(j+1+"/")+folderCount ;
+            emitter.emit("SHOW_PROGRESS_BAR", msg);
+            emitter.emit("PROGRESS_BAR_WIDTH", (((j+1)/(overallitems))*100));
             }
```

Comment:
this brace is indented too much move it back one step (it's not aligned with the `for` statement)